kratos/python_scripts/point_output_process.py

```diff
@@ -59,6 +60,13 @@ def __init__(self, model, params):
         self.historical_value = self.params["historical_value"].GetBool()
         self.search_tolerance = self.params["search_tolerance"].GetDouble()
 
+        if self.params["search_configuration"].GetString() == "Initial":
+            self.search_configuration = KratosMultiphysics.Configuration.Initial
+        elif self.params["search_configuration"].GetString() == "Current":
+            self.search_configuration = KratosMultiphysics.Configuration.Current
+        else:
+            raise Exception( "Invalid configuration: %s (Expecting 'Initial' or 'Current')" % self.params["search_configuration"].GetString() )
```

Comment:
You don't need to change, but I recommend to take a look at the `format` method (and `f-strings`, from python3.6) for your own understanding